<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_LineAngleOptions</class>
 <widget class="QWidget" name="Ui_LineAngleOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>22</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>22</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Line Angle Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Angle:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="cbSnapPoint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font/>
     </property>
     <item>
      <property name="text">
       <string>Start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Line angle</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="leLength">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Length of line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lLength">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lSnapPoint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Snap Point:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <resources/>
 <connections>
  <connection>
   <sender>leAngle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Ui_LineAngleOptions</receiver>
   <slot>updateAngle(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLength</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Ui_LineAngleOptions</receiver>
   <slot>updateLength(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSnapPoint</sender>
   <signal>activated(int)</signal>
   <receiver>Ui_LineAngleOptions</receiver>
   <slot>updateSnapPoint(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateAngle(QString)</slot>
  <slot>updateLength(QString)</slot>
  <slot>updateSnapPoint(int)</slot>
 </slots>
</ui>
